look-and-feel/react/src/List/ContentTabList/ContentTabItem/ContentTabItem.tsx

```diff
@@ -61,9 +61,9 @@ export const ContentTabItem = ({
             )}
           </div>
         )}
-        {buttons.map((button) => (
-          <div className="af-list-item__button-container" key={button?.id}>
-            <Button {...button} />
+        {buttons.map(({ id, component }) => (
```

Comment:
peut-être renommer la prop si ce n'est plus forcément un button

Reply:
en `actions` peut-être ?

Reply:
Fixed in : https://github.com/AxaFrance/design-system/pull/589/commits/5a893c921e8535ba9f76dcbe350ceaf05e2ec52e
(je te laisse close si le nom `actions` te convient)